src/styles/blocks/header.scss

```diff
@@ -0,0 +1,259 @@
+      font-size: 40px;
+      line-height: 40px;
+      padding-bottom: 40px;
+    }
+
+    @media (min-width: 351px) and (max-width: 400px) {
+      font-size: 50px;
+      line-height: 50px;
+      padding-bottom: 50px;
+    }
+
+    @media (min-width: 400px) and (max-width: 576px) {
+      font-size: 50px;
+      line-height: 50px;
+      padding-bottom: 50px;
+    }
+
+    @media (min-width: 450px) and (max-width: 576px) {
+      font-size: 60px;
+      line-height: 60px;
+      padding-bottom: 60px;
+    }
+
+    @media (min-width: 577px) and (max-width: 768px) {
+      padding-bottom: 40px;
+    }
+
+    &::before {
+      content: "";
+      display: block;
+      width: 40px;
+      height: 2px;
+      background: #fff;
+      border-radius: 2px;
+      margin-bottom: 24px;
+    }
+  }
+}
+
+.nav,
+.logo,
+.phone {
+  width: 33.33%;
+}
+
+.nav {
+  &__icon {
+    background-image: url(../images/icons/menu-icon.svg);
+    background-repeat: no-repeat;
+    position: relative;
+    z-index: 1;
+    width: 24px;
+    height: 16px;
+
+    transition: background-image 400ms;
+
+    @media (min-width: 100px) and (max-width: 576px) {
+      width: 18px;
+      height: 16px;
+    }
+
+    &:hover {
+      background-image: url(../images/icons/menu-icon-hover.svg);
+      background-repeat: no-repeat;
+      background-size: cover;
+      cursor: pointer;
+      width: 24px;
+      height: 16px;
+
+      @media (min-width: 100px) and (max-width: 576px) {
+        width: 18px;
+        height: 16px;
+      }
+    }
+  }
+
+  &__menu {
+    display: none;
+    align-items: flex-start;
+
+    position: absolute;
+    width: 100%;
+    height: 700px;
+    left: 0;
+    top: 0;
+    padding-left: 80px;
+
+    background-color: #0075ff;
+
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 22px;
+    color: #fff;
+
+    transition: background-color 400ms;
+
+    @media (min-width: 100px) and (max-width: 576px) {
+      padding-left: 20px;
+    }
+
+  }
+
+  &__items {
+    padding-top: 152px;
+  }
+
+  &__item {
+
+    text-transform: uppercase;
+    margin-bottom: 32px;
+    transition: display, padding-bottom, border-bottom, 400ms;
+
+    &:hover {
+      display: inline-block;
+      border-bottom: 2px solid #fff;
+      padding-bottom: 8px;
+      margin-bottom: 24px;
+      cursor: pointer;
+    }
+
+    &--phone {
+      font-size: 12px;
+      padding: 16px 0 0;
+      margin-bottom: 0;
+
+      &:hover {
+        border-bottom: 1px solid #fff;
+        width: 100px;
+        margin-bottom: 0;
+        padding-bottom: 0;
+        cursor: pointer;
+      }
+    }
+
+    &--text {
+      font-size: 12px;
+      text-transform: uppercase;
+      padding-top: 16px;
+      
+      &:hover {
+        width: 90px;
+        border-bottom: 1px solid #fff;
+        margin-bottom: 0;
+        padding-bottom: 0;
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+.logo {
+  width: 136px;
```

Comment:
You wrote before

```
.nav,
.logo,
.phone {
  width: 33.33%;
}
```
It's conflicting rules here

Reply:
(actually, there's no need to specify these elements width because you wrote `justify-content: space-between`)